doc/design/fixedString.adoc

```diff
@@ -31,157 +31,3 @@ In the second step, the functionality has been extended:
 .step 3:
```

Comment:
Could we remove this file completely? A design document for the fixed string does not make sense in my opinion.
If we would require this that we would clutter the doc/design/ folder with a design document for every cxx container and construct.

Reply:
we need maybe one where we describe our intention with the re-implementation of the STL container, what constraints our container have and how they deviate from the STL

Reply:
This already described [here](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_utils#cxx).